src/index.html

```diff
@@ -6,8 +6,58 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+
+      <img
+        src="images/imac.jpeg"
+
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+
+        >
+
+      <h2 class="card__info">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <section class="card__code">Product code: 195434</section>
+
+      <section class="card__reviws">
+        <div class="stars stars--4">
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
+          <mark class="stars__star"></mark>
```

Comment:
The <mark> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents text which is marked or highlighted for reference or notation purposes due to the marked passage's relevance in the enclosing context.

so let's use a div tag here